.github/workflows/docker-publish-action.yml

```diff
@@ -1,19 +1,22 @@
-name: Publish API/UI Docker Image
-on: workflow_dispatch
+on:
+  workflow_call:
+    inputs:
+      IMAGE_NAME:
+        required: true
+        type: string
+      DOCKERFILE:
+        required: false
+        type: string
 
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME_UID2: iabtechlab/uid2-ssportal
 
 jobs:
   build-publish-docker:
     runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      packages: write
```

Comment:
another naïve question: why don't we need this anymore? Are we just using defaults instead and prefer that?

Reply:
or is it because the permissions are specified now in `docker-publish-ssportal.yml`?

Reply:
Yes, I put it in there